BazyDanych/BazaDanych/BazaDanych/Model/Book.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace BazaDanych.Model
+{
+    internal class Book
+    {
+        public Book(string name, int departmentId)
+        {
+            Name = name;
+            DepartmentId = departmentId;
+        }
+
+        public int BookId { get; set; }
```

Comment:
SUG: Jak przeładowujesz metodę ToString w modelach to przeładuj ją we wszystkich.

Reply:
[022b447](https://github.com/broniuu/Praktyki2022/pull/3/commits/022b447cf58602ada368ad586d2a93ac13e7e970)